.pipelines/azure_pipeline_mergedbranches.yaml

```diff
@@ -27,12 +27,11 @@ jobs:
     name: Azure-Pipelines-CI-Test-EO
   steps:
   - task: ComponentGovernanceComponentDetection@0
-
+  ## commit will be in format Major.Minor.Patch-CommitAheadCount-CommitId
   - bash: |
-      commit=$(git rev-parse --short HEAD)
-      datetime=$(date +'%m%d%Y')
-      linuxImagetag="$datetime"-"$commit"
-      windowsImageTag=win-"$datetime"-"$commit"
+      commit=$(git describe) 
```

Comment:
can you respond what the tags will look like for 

1. prod release
2. our dev builds
3. preview prod release

Reply:
added in comments above